### ISSUE TITLE
Bootstrap throws errors in safari

### ISSUE THREAD

**Issue by jteplitz602** [angular/angular#3015]
To repro: Load the hello world example in safari.
Stack trace: https://gist.github.com/jteplitz602/26f330e39b60d99a69d3

**Comment by justindujardin** [comment]
This is also reproduced in my application.  https://justindujardin.github.io/angular2-rpg/

```
[Error] TypeError: undefined is not a function (evaluating 'y.next()')

t@https://github.jspm.io/jmcriffey/bower-traceur-runtime@0.0.87/traceur-runtime.js:3:6331
values@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:8092:30
_createListOfBindings@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:26967:29
resolve@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:27564:41
resolveAndCreate@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:27568:52
_createAppInjector@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:29780:48
http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:29761:43
run@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:118:22
http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:13938:43
run@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:13905:41
bootstrap@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:29760:13
http://justindujardin.github.io/angular2-rpg/:80:33
http://justindujardin.github.io/angular2-rpg/bower_components/pow-core/lib/pow-core.js:161:31
http://justindujardin.github.io/angular2-rpg/bower_components/underscore/underscore-min.js:5:8167
triggerEvents@http://justindujardin.github.io/angular2-rpg/bower_components/pow-core/lib/pow-core.js:255:51
trigger@http://justindujardin.github.io/angular2-rpg/bower_components/pow-core/lib/pow-core.js:214:30
http://justindujardin.github.io/angular2-rpg/:48:37
run@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:118:22
zoneBoundFn@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:91:22
lib$es6$promise$$internal$$tryCatch@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:1409:24
lib$es6$promise$$internal$$invokeCallback@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:1421:52
lib$es6$promise$$internal$$publish@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:1392:52
http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:169:7
run@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:118:22
zoneBoundFn@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:91:22
lib$es6$promise$asap$$flush@http://justindujardin.github.io/angular2-rpg/vendor/angular2/angular2.dev.js:1203:17
	logError (angular2.dev.js, line 20718)
	defaultErrorReporter (angular2.dev.js, line 29746)
	_onError (angular2.dev.js, line 13981)
	onError (angular2.dev.js, line 13924)
	run (angular2.dev.js, line 121)
	(anonymous function) (angular2.dev.js, line 13938)
	run (angular2.dev.js, line 13905)
	bootstrap (angular2.dev.js, line 29760)
	(anonymous function) (angular2-rpg, line 80)
	(anonymous function) (pow-core.js, line 161)
	(anonymous function) (underscore-min.js, line 5)
	triggerEvents (pow-core.js, line 255)
	trigger (pow-core.js, line 214)
	(anonymous function) (angular2-rpg, line 48)
	run (angular2.dev.js, line 118)
	zoneBoundFn (angular2.dev.js, line 91)
	lib$es6$promise$$internal$$tryCatch (angular2.dev.js, line 1409)
	lib$es6$promise$$internal$$invokeCallback (angular2.dev.js, line 1421)
	lib$es6$promise$$internal$$publish (angular2.dev.js, line 1392)
	(anonymous function) (angular2.dev.js, line 169)
	run (angular2.dev.js, line 118)
	zoneBoundFn (angular2.dev.js, line 91)
	lib$es6$promise$asap$$flush (angular2.dev.js, line 1203)
```

**Comment by mlaval** [comment]
Found the root cause of the issue.

In Safari, `MapIterator` doesn't implement `next()`,
which is used in the traceur-runtime's polyfill of `Array.from`,
which is used in `MapWrapper.keys()` and `MapWrapper.values()` in Angular2's facade.
This makes about 860 unit tests to fail.

Not sure if we should try to fix Traceur knowing that it should go away, or if we should work around this bug. Any thoughts please?

**Comment by hafez** [comment]
It appears the issue has been addressed in latest version of traceur-runtime/0.0.91 (was previously using version 0.0.89 and experienced the same issue when running under Safari - runs fine with latest traceur).
